x-pack/plugins/apm/server/lib/helpers/create_es_client/create_infra_metrics_client/create_infra_metrics_client.ts

```diff
@@ -9,7 +9,12 @@ import { ESSearchRequest, InferSearchResponseOf } from '@kbn/es-types';
 import { APMRouteHandlerResources } from '../../../../routes/typings';
 import { getInfraMetricIndices } from '../../get_infra_metric_indices';
 
-type InfraMetricsSearchParams = Omit<ESSearchRequest, 'index'>;
+type InfraMetricsSearchParams = Omit<ESSearchRequest, 'index'> & {
+  body: {
```

Comment:
Do you really need the `body` here?

Reply:
I was following @sqren suggestion here: https://github.com/elastic/kibana/pull/141735#discussion_r983991233